src/installer.py

```diff
@@ -111,48 +105,28 @@ def uninstall(self, password):
         
         #Starting bash and getting root privileges
         print("Starting bash and getting root privileges")
-        pyotherside.send('whatState',"=> Starting uninstaller")
-        time.sleep(1.5)
+        pyotherside.send('state', 'starting', False)
         child = pexpect.spawn('bash')
+        child.logfile_read = sys.stdout.buffer
         child.expect(r'\$')
         child.sendline('sudo -s')
         if password != '':
             child.expect('[p/P]ass.*')
             child.sendline(str(password))
         child.expect('root.*')
 
-        #remounting filesystem to rw
-        print("remounting filesystem to rw")
-        pyotherside.send('whatState',"=> remounting filesystem as readwrite")
-        time.sleep(1.5)
-        child.sendline('mount -o remount,rw /')
-        child.expect('root.*')
-
         #Stop Waydroid Container
         print("stopping waydroid container")
-        pyotherside.send('whatState',"=> stopping waydroid container")
+        pyotherside.send('state', 'container', False)
         child.sendline("service waydroid-container stop")
```

Comment:
I suppose this already works but should become `systemctl stop waydroid-container` (same with other `service` commands if any more present)

Reply:
Aren't we going to use dbus? And the 1.4 version of Waydroid is closer

Reply:
> Aren't we going to use dbus? And the 1.4 version of Waydroid is closer

Reply:
D-Bus to do what? Starting/stopping the container over D-Bus is a bit pointless for now since that just in turn calls the `systemd` service (does it even actually do that?)

Reply:
Systemd has it's own dbus interface.

Ayway dbus in waydroid stops and starts waydroid  same way it would stop and restart from the cmdline. Also would be helpful later if for example you want to restart waydroid for some reason

Reply:
It's not a user service, so we'd need root anyway (iirc even if we had a polkit agent in UT systemd doesn't support it). So it would have to run some other python script as root, doesn't make much sense to re-implement systemctl.

Reply:
> It's not a user service, so we'd need root anyway (iirc even if we had a polkit agent in UT systemd doesn't support it). So it would have to run some other python script as root, doesn't make much sense to re-implement systemctl.

Oh good point. It's a system service